engine/common/cvar.c

```diff
@@ -887,6 +998,40 @@ void Cvar_Toggle_f( void )
 	Cvar_Set( Cmd_Argv( 1 ), va( "%i", v ));
 }
 
+/*
+============
+Cvar_Set_f
+
+Allows setting and defining of arbitrary cvars from console, even if they
+weren't declared in C code.
+============
+*/
+void Cvar_Set_f( void )
+{
+	int	i, c, l = 0, len;
+	char	combined[MAX_CMD_TOKENS];
+
+	c = Cmd_Argc();
+	if( c < 3 )
+	{
+		Msg( "Usage: set <variable> <value>\n" );
```

Comment:
We can use S_USAGE macro here:
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/com_strings.h#L23=